<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Zafar" failures="0" tests="1" name="test4" time="3.818" errors="0" timestamp="2023-02-27T15:05:45 IST">
  <testcase classname="com.tyss.genricUtility.ConfigerClass.MainClass" name="@BeforeMethod loginApp" time="0.654">
    <failure type="java.lang.AssertionError" message="Home Page is Open Successfully expected [Home] but found [Login Page]">
      <![CDATA[java.lang.AssertionError: Home Page is Open Successfully expected [Home] but found [Login Page]
at com.tyss.genricUtility.ConfigerClass.MainClass.loginApp(MainClass.java:86)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod loginApp -->
  <testcase classname="com.tyss.genricUtility.ConfigerClass.MainClass" name="@AfterMethod LogoutApp" time="-1.677490544918E9">
    <skipped/>
  </testcase> <!-- @AfterMethod LogoutApp -->
  <testcase classname="importantTestCase.Client_Add_Payment" name="client_Add_Payment" time="0.0">
    <skipped/>
  </testcase> <!-- client_Add_Payment -->
</testsuite> <!-- test4 -->
